optuna/visualization/_pareto_front.py

```diff
@@ -96,6 +97,14 @@ def objective(trial):
             If given, trials are classified into three categories: feasible and best, feasible but
             non-best, and infeasible. Categories are shown in different colors. Here, whether a
             trial is best (on Pareto front) or not is determined ignoring all infeasible trials.
+        targets:
+            A function that returns a tuple of target values to display.
```

Comment:
```suggestion
            A function that returns target values to display.
```

`targets` returns `Sequence` of float. We need to update matplotlib as well. 💭